source/whats-new.txt

```diff
@@ -0,0 +1,34 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+
+* :ref:`Version 2.19 <version-2.19>`
+
+.. _upcoming-breaking-changes:
+
+.. _version-2.19:
+
+What's New in 2.19 
+-------------------
+
+New features of the 2.19 driver release include:
+
+- The ``ObjectSerializer`` allows deserialization of only safe types, as determined by 
+  the configurable ``AllowedTypes`` function. 
+- Changed the default ``LinqProvider`` to LINQ3.
+- Added builders for creating Atlas searches.
+- Added support for Range Indexes preview.
+- Added support for Bucket and BucketAuto stages in LINQ3.
```

Comment:
[Question] Should Bucket and BucketAuto be stylized as `code`? I honestly don't know.

Reply:
I had the same question, but left them as is because they're stages. It looks like we do `$bucket` in our server documentation, so I'll do the same here.